pages/design-tokens/color/overview.md

```diff
@@ -179,15 +179,15 @@ The following image shows our vivid System tokens wheel.
 
 If we use color intentionally, consistently, and sensitively, it can make a big difference in the way people understand and connect with our pages, our products and services, and our message. Color is an important component of visual and emotional cognition, and that’s precisely what makes it difficult to use well — what’s strong and confident to one person can be jarring or alarming to another.
 
-**Start in black and white.** Start with your core message and use type scale and hierarchy to test and refine its effectiveness. Then, introduce color to support that message. Color can overwhelm interpretation, and since [approximately 4.5% of the population](https://nei.nih.gov/health/color_blindness/facts_about) has some kind of color insensitivity, it’s important to convey your message without relying on color.
+**Start in black and white.** Start with your core message and use type scale and hierarchy to test and refine its effectiveness. Then, introduce color to support that message. Color can overwhelm interpretation, and since [approximately 4.5% of the population](https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness) has some kind of color insensitivity, it’s important to convey your message without relying on color.
```

Comment:
**question:** I can't find a reference to the percent of the general population with color blindness on this new page. Will that be an issue for keeping this as a reference?

Reply:
Removing the reference here, since it's cited above and there are now two separate NIH references

Reply:
```suggestion
**Start in black and white.** Start with your core message and use type scale and hierarchy to test and refine its effectiveness. Then, introduce color to support that message. Color can overwhelm interpretation, and since approximately 4.5% of the population has some kind of color insensitivity, it’s important to convey your message without relying on color.
```